### PR TITLE
Fix unique

### PR DESCRIPTION
**Pull request by raimis**
Depend on https://github.com/Acellera/htmd/pull/793

### CODE REVIEW

File: test-param/H2O2.mol2 [hunk 1]
```diff
@@ -10,6 +10,6 @@ GASTEIGER
       3 Hc          0.8212   -0.8236   -0.4306 H       1  UNL1        0.2528
       4 Hd          2.6555    0.7137    0.4887 H       1  UNL1        0.2528
 @<TRIPOS>BOND
-     1     1     2    1
-     2     1     3    1
-     3     2     4    1
+     1     1     2    un
```
Comment (j3mdamas) [inline test-param/H2O2.mol2]: Why did you change this?

Comment (raimis) [reply test-param/H2O2.mol2]: To make sure, that the bond type detection actually works!
